jx_sqlite/aggs_table.py

```diff
@@ -205,9 +205,15 @@ def _get(row):
             elif len(edge_names) > 1:
                 domain_names = ["d" + text_type(edge_index) + "c" + text_type(i) for i, _ in enumerate(edge_names)]
                 query_edge.allowNulls = False
+                domain_columns = [c for c in self.sf.columns if quote_table(c.es_column) in vals]
+                if domain_columns and path==domain_columns[0].nested_path[0]:
+                    domain_table = quote_table(frum)
+                else:
+                    domain_table = quote_table(self.sf.fact)
+
```

Comment:
I have not run this yet, but I am lost as to what this is doing.  Specifically, why would `quote_table(self.sf.fact)` ever be preferred over `quote_table(frum)`?  It seems odd to override `frum`. It looks like another special case (and it is duplicated in another hunk below).  If this special case is important (and it is not indicating a problem in some other part of the code), should this be a function?  

I understand the old code; `quote_table(self.sf.fact)` it was wrong for the sake of simplicity.